plugins/brimcap/brimcap-plugin.ts

```diff
@@ -24,7 +24,7 @@ export default class BrimcapPlugin {
   private selectedConn = null
   private yamlConfigPath = ""
   private cleanupFns: Function[] = []
-  private loadingProcesses: {
+  private processes: {
```

Comment:
reusing this as just a dump of any async processes, so that I can also reuse the cleanup logic invoked when users quit or refresh the app